scripts/main_app.py

```diff
@@ -0,0 +1,15 @@
+import streamlit as st
+from pages import demo_playbooks, playbook_builder
+from streamlit import session_state as ss
+
+if "current_page" not in st.session_state:
```

Comment:
@pavangudiwada we're inconsistent in this file. Sometimes we set session_state via `ss` and sometimes we do it via `st.session_state` - can you choose one and use it consistently.

Reply:
Done :heavy_check_mark: